portals/publisher/src/main/webapp/source/src/app/components/Shared/PoliciesUI/AttachedPolicyCard.tsx

```diff
@@ -106,70 +109,153 @@ const AttachedPolicyCardShared: FC<AttachedPolicyCardSharedProps> = ({
         backgroundColor: policyBackgroundColor,
         opacity: isDragging ? 0.5 : 1,
     };
-    return (
-        (<Root>
-            <div
-                ref={setNodeRef}
-                style={style}
-                {...attributes}
-                {...listeners}
-                onClick={handleDrawerOpen}
-                onKeyDown={handleDrawerOpen}
-            >
-                <Tooltip
-                    key={policyObj.id}
-                    title={`${policyObj.displayName} : ${policyObj.version}`}
-                    placement='top'
+
+    if ('listOriginatedFromCommonPolicies' in props) {
```

Comment:
Can't we specify this as a prop passed down from the parent component instead?

Reply:
relates to https://github.com/wso2/apim-apps/pull/809#discussion_r1901575381